Rakefile

```diff
@@ -295,6 +295,8 @@ begin
 
             project = Xcodeproj::Project.open(project_path)
             target = project.targets.first
+            scheme_target = project.targets.find { |t| t.name == scheme_name }
+            target = scheme_target unless scheme_target.nil?
```

Comment:
we were always using the first target to determine the platform, which doesn't work for projects that have multiple targets with different platforms